src/main/groovy/com/blackducksoftware/integration/hub/detect/DetectProjectManager.groovy

```diff
@@ -139,7 +140,10 @@ class DetectProjectManager implements SummaryResultReporter, ExitCodeReporter {
 
         if (!foundAnyBomTools) {
             logger.info("No package managers were detected - will register ${detectConfiguration.sourcePath} for signature scanning of ${detectProject.projectName}/${detectProject.projectVersionName}")
-            hubSignatureScanner.registerPathToScan(detectConfiguration.sourceDirectory)
+            hubSignatureScanner.registerPathToScan(ScanPathSource.DETECT_SOURCE, detectConfiguration.sourceDirectory)
+        } else if (detectConfiguration.hubSignatureScannerSnippetMode) {
```

Comment:
This is all that is needed to support Hal's feature request now that we've made the other changes - I think this location makes sense.